connectivity/lorawan/lorastack/phy/LoRaPHYAS923.cpp

```diff
@@ -337,8 +411,8 @@ int8_t LoRaPHYAS923::get_alternate_DR(uint8_t nb_trials)
 }
 
 lorawan_status_t LoRaPHYAS923::set_next_channel(channel_selection_params_t *next_channel_prams,
-                                                uint8_t *channel, lorawan_time_t *time,
-                                                lorawan_time_t *aggregate_timeoff)
+        uint8_t *channel, lorawan_time_t *time,
```

Comment:
this requires to be edited as style check is failing, see

```
diff --git a/connectivity/lorawan/lorastack/phy/LoRaPHYAS923.cpp b/connectivity/lorawan/lorastack/phy/LoRaPHYAS923.cpp
index 399a5c2b90..41d7053053 100644
--- a/connectivity/lorawan/lorastack/phy/LoRaPHYAS923.cpp
+++ b/connectivity/lorawan/lorastack/phy/LoRaPHYAS923.cpp
@@ -411,8 +411,8 @@ int8_t LoRaPHYAS9[23](https://github.com/ARMmbed/mbed-os/runs/7873972255?check_suite_focus=true#step:5:24)::get_alternate_DR(uint8_t nb_trials)
 }
 
 lorawan_status_t LoRaPHYAS923::set_next_channel(channel_selection_params_t *next_channel_prams,
-        uint8_t *channel, lorawan_time_t *time,
-        lorawan_time_t *aggregate_timeoff)
+                                                uint8_t *channel, lorawan_time_t *time,
+                                                lorawan_time_t *aggregate_timeoff)
 {
     uint8_t next_channel_idx = 0;
     uint8_t nb_enabled_channels = 0;
@@ -436,8 +436,8 @@ lorawan_status_t LoRaPHYAS923::set_next_channel(channel_selection_params_t *next
 
         // Search how many channels are enabled
         nb_enabled_channels = enabled_channel_count(next_channel_prams->current_datarate,
-                              channel_mask,
-                              enabled_channels, &delay_tx);
+                                                    channel_mask,
+                                                    enabled_channels, &delay_tx);
     }  else {
         delay_tx++;
```

Reply:
Applied the astyle patch from the style check.
Don't understand what happened before as i run `astyle` command as instructed.